### ISSUE TITLE
Plain HTTP channel created through public API appears as SOAP channel in service view

### ISSUE THREAD

**Issue by pedrudehuere** [zatosource/zato#466]
A tiny one.

When creating a plain HTTP channel with the public API, and then clicking on the concerned service, the created channel appears under "Channels and scheduler" as SOAP channel.
Clicking on the channel opens the SOAP channel page, the channel is not displayed (which is good since it is not a SOAP channel). The channel appears in the plain HTTP channels page though.
The channel works fine, it should just be a matter of display.
This does not happen for channels created with the web-admin.

**Comment by dsuch** [comment]
Thanks @pedrudehuere - soon will be seen to.

**Comment by dsuch** [comment]
Hi @pedrudehuere,

I've just tried it on 2.0.5 and I cannot reproduce it. I used ```curl``` to submit the following JSON document to the ```zato.http-soap.create``` service:

```json
{
"sec_tls_ca_cert_id": null,
"is_internal": false,
"is_active": true,
"cluster_id": 1,
"has_rbac": false,
"url_path": "/test",
"transport": "plain_http",
"ping_method": null,
"name": "test",
"service": "zato.ping",
"soap_version": null,
"data_format": "",
"url_params_pri": "qs-over-path",
"connection": "channel",
"merge_url_params_req": true,
"security_id": "ZATO_NONE",
"params_pri": "channel-params-over-msg",
"method": ""
}
```

Everything works as expected. The channel was created and it gets listed along other plain HTTP channels, i.e. it's not one of the SOAP ones.

Could you please tell me which Zato version you are on and what the document you submit is?

Thanks a lot.

**Comment by pedrudehuere** [comment]
you are right, with those data it works fine,
but if "soap_version" is missing or if its value is an empty string then the error occurs.

I'm on Zato 2.0.5

**Comment by dsuch** [comment]
Thanks @pedrudehuere - fixed in 909426a2c14f3d45fa8eea2704a43eddbac0cb37 and will be included in 2.0.6.

**Comment by pedrudehuere** [comment]
Thank you =)
